config/StoredProcedures/GetCustomers.sql

```diff
@@ -0,0 +1,13 @@
+
+use imp_db_dev;
+DROP PROCEDURE IF EXISTS GetCustomers;
+
+DELIMITER $$
+CREATE PROCEDURE GetCustomers()
```

Comment:
this might be overly picky, but can we rename this GetCustomerIDs? GetCustomers seems like it would get more information from the table than just their IDs